src/main/java/org/openrewrite/java/migrate/jakarta/UpdateBeanManagerMethod.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.openrewrite.java.migrate.jakarta;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.jetbrains.annotations.NotNull;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Option;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.internal.lang.NonNull;
+import org.openrewrite.internal.lang.Nullable;
+import org.openrewrite.java.JavaTemplate;
+import org.openrewrite.java.JavaVisitor;
+import org.openrewrite.java.MethodMatcher;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.JavaType;
+import org.openrewrite.java.JavaParser;
+
+public class UpdateBeanManagerMethod extends Recipe {
+    @Option(displayName = "Method Pattern",
+            description = "A `BeanManager.fireEvent()` or `BeanManager.createInjectionTarget()` matching required",
+            example = "jakarta.enterprise.inject.spi.BeanManager fireEvent()")
+    @NonNull String methodPattern;
+
+    @Override
+    public @NotNull String getDisplayName() {
```

Comment:
Please remove excess annotations like this one here as well as in other places;
Not sure where they are coming from, but they don't make sense here.
```suggestion
    public String getDisplayName() {
```

Reply:
@ranuradh Could you address this feedback still?

Reply:
@timtebeek addressed this in the next commit